.editorconfig

```diff
@@ -0,0 +1,716 @@
+[*]
```

Comment:
One suggestion add:

`root = true` to the start of the file.

Reply:
Thanks 👍 , done.